src/components/MessageActions/MessageActions.tsx

```diff
@@ -52,6 +55,7 @@ export const MessageActions = <
   props: MessageActionsProps<At, Ch, Co, Ev, Me, Re, Us>,
 ) => {
   const {
+    ActionsIcon,
```

Comment:
you can set default value here to DefaultActionsIcon and then delete line 95 and rename 140 to ActionsIcon